cl/users/templates/includes/webhooks_htmx/webhooks-pagination.html

```diff
@@ -0,0 +1,40 @@
+{% load humanize %}
+{% if results.has_previous or results.has_next %}
+  <div class="row v-offset-above-1">
+    <div class="col-md-3"></div>
+    <div class="col-xs-2 col-sm-2">
+      {% if results.has_previous %}
+        <div class="text-left">
+          <button id="webhooks-paginator"
```

Comment:
Can this pagination file be merged with the normal include, so it's not so much duplicated code? I guess you tried that, but it wasn't worth it?

Reply:
Yeah, now using the global pagination after merging the webhook pagination variants:
![Screen Shot 2022-11-21 at 11 15 24](https://user-images.githubusercontent.com/486004/203128959-55dd5e07-3b61-46f0-bf7f-54b56bbb1842.png)